### ISSUE TITLE
dragThreshold withing breakpoints possibly broken

### ISSUE THREAD

**Issue by robbinfellow** [glidejs/glide#293]
👋 

Great plugin, love it! 

**Browsers:** 
- Chrome Version 69.0.3497.100 (Official Build) (64-bit)
- Safari Version 12.0 (14606.1.36.1.9)

**OS:**
Mac os Mojave

**Glidejs version:**
@glidejs/glide@3.2.3

I ran into an issue when trying to modify `dragThreshold` on different breakpoints.

When resizing the browser window, the `dragThreshold` setting  is updated correctly, if i `console.log` the settings of `dragThreshold` I can see that it has been updated, however it doesnt reflect on the slider itself, unless I refresh the browser.

Using glide in a vue component:

``` javascript
this.glide = new Glide(this.$refs.glide, {
  type: 'slider',
  perView: 2,
  rewind: false,
  keyboard: false,
  dragThreshold: false,
  gap: '0',
  animationDuration: 600,
  animationTimingFunc: 'ease',
  breakpoints:
  {
    767: {
      peek: { before: 0, after: 80 },
       perView: 1,
      dragThreshold: 120,
    }
  },
}).mount();
```

TLDR; 
When I resize the browser window to 767 from a larger screen size I can't drag the slider, and if I resize the browser window from `> 767` and up I **can** drag the slider even though the settings says different.

Below is a screenshot from logging the dragThreshold on resize, as you can see it updates correctly but isn't actually working in practice:

<img width="98" alt="screenshot 2018-11-01 at 08 30 06" src="https://user-images.githubusercontent.com/23704274/47838706-7d3e0800-ddb0-11e8-8777-9f57085d4a3b.png">
